README.md

```
@@ -1,207 +1,266 @@
-<img height="200px" src="./assets/docs/app_icon.png" align="right" />
+<img height="200px" src="./assets/icons/app_icon_desktop.png" align="right" />
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)